lib/torrent.js

```diff
@@ -893,7 +893,7 @@ Torrent.prototype.deselect = function (start, end, priority) {
   for (var i = 0; i < self._selections.length; ++i) {
     var s = self._selections[i]
     if (s.from === start && s.to === end && s.priority === priority) {
-      self._selections.splice(i--, 1)
+      self._selections.splice(i, 1)
```

Comment:
the `--` is not needed to compensate for the `splice()`, since we call `break` right afterwards.